### ISSUE TITLE
Propose styles for header, footer, navigation, form, and content components 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#392]
Please review the styles I have proposed: there are [4 versions here](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=163%3A6825): 

Note: 
- these screenshots may not be clear – it's probably best to open the link above and zoom out to see the large headings I have added to help you navigate the Figma page. 
- also, look for the following throughout the pages: 1. separator styles (between each item in the search results), 2. new bullet styles on the content page, 3. new quotation mark styles for the pull quotes on the content page 4. decorative Hebrew text overlay on header and footer (we would need to pick a meaningful piece of text that's displayed correctly for this kind of design) 4. decorative /// (dash) element in header, 5. hover styles in header (the squiggly line, the straight line, and the dot styles)
- I'm not sure if we can place Hebrew text overlay in the transcription/image pane but if we can and we want to do it: it would need to either be a static piece of text that could be displayed for all documents or a piece of text that is taken from the transcription being displayed so it's contextual and meaningful 

v1: 
![Screen Shot 2021-12-04 at 12.12.35 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/11189113-8d7e-4345-a577-81bdb8dc890b)

v2: 
![Screen Shot 2021-12-04 at 12.13.02 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/ec32123f-4963-4cb1-bec4-37bc2ae34a31)
![Screen Shot 2021-12-04 at 12.13.41 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/9956389d-b874-42c1-9abd-de58f3dcb750)

v3:
![Screen Shot 2021-12-04 at 12.14.45 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/2fb0ec88-a332-48f3-b094-6a6ead7aea27)

v4: 
![Screen Shot 2021-12-04 at 12.15.07 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/7e35db7e-ad7d-4f3a-b938-635b886ca090)

Dark Mode: 
![Screen Shot 2021-12-04 at 12.15.59 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/4b4cfbd7-cb67-4228-8969-834eda4327d4)

Red or Green buttons: (I preferred the red, but wanted to make sure we see more options) 
![Screen Shot 2021-12-04 at 12.17.44 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/2ff5ac14-3693-4cdf-9a99-bc29a0cbfc25)
![Screen Shot 2021-12-04 at 12.19.13 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/3028bd9f-a082-4f1b-be21-fd4e00dd2f43)

**Comment by richmanrachel** [comment]
@gissoo - I think V3 is my favorite, followed by V1. 

I agree with your preference for the red buttons.

I don't think I like the red background with white text for the transcription... Even though my brain does go into a different mode to read Judaeo-Arabic, having to do the additional gymnastics of switching font color (and worrying about it not copy-pasting into black) seems like it might be stressful. 

1) I like the V1 inked line, but I would drop using a Hebrew word here to make it less complicated and more clear that it's a separator and not content.

2) I'm ambivalent about the new bullet styles on the content page - I probably would get used to them? They do look a little like one of the letters in Judaeo-Arabic, I don't know if that's a good thing or confusing :) 

3) Quotation marks are fun. 
 
4) I like the way the decorative Hebrew on header and footer looks. I don't have a text in mind.

5) I don't feel strongly about the decorative elements in the header

6) I prefer the straight line for the hover style in header. Squiggly line in second place, the dot is too understated to clearly interpret.

**Comment by rlskoeser** [comment]
I like the header/footer design in V2 best. I like V3 as well. I think the colors/shadows for V2 in dark mode need to be adjusted if we go with that one, it doesn't read the same as the light mode version in terms of overlap.

I don't like the curled up page corner versions. (I feel like it has been used before... also, what does it suggest? what is behind it? is there something clickable and visible underneath?)

I like the underline best for header selection; I like the squiggly line but it seems a little bit too much; the dot doesn't seem like enough of a marker.

**Comment by blms** [comment]
I agree with Rebecca and Rachel's comments. In addition:

- I find the "torn fragment" header/footer design to be overly ornate, but I tend to lean towards more minimal designs.
- I really like the way you've styled the navigation tabs (document detail/scholarship records/external links).
- I like the rightmost v1 "inked line" separator between search results. The three dots from v2-v4 could also work for me if they were altered to look less clickable/interactive, as right now they might convey an action (e.g. read more/expand).

**Comment by rlskoeser** [comment]
I never commented on the separator styles! Sorry.

I like the long line that tapers off the best. Alternating styles makes sense for search results but I don't know if it would work as well elsewhere, like in a content document; but maybe it isn't intended for that?

I also like the three dots in a row (second choice); I don't like the three dots making a triangle as well.

I like the /// version (esp. if we go with something that incorporates that in the header), but feel like it fades out too quickly.

**Comment by gissoo** [comment]
Thank you all for your comments and our conversations
Documenting our decisions:
- header and the footer: we went with the reversed fragmented style (v2) and removed the solid color rectangle from the bottom layer to simplify it. 
- separator styles: we chose the long line + decided to use the alef in the Hebrew alphabet with a long line as the separator between the pagination and the first result only on the first page of the search results
- hover style in site menu: we chose the squiggly line, which will be drawn differently to add some variety to the thickness of the line
- bullet style: decided to use the proposed bullet style I had drawn + added color to it. 
- quotation marks for pull quotes: decided to use the revised version I created
- header and footer overlay style: already revised it based on the fragment images that Marina sent and I edited.
- decided to omit the dark solid color behind the transcription section of the detail view and revised it to a lighter grey, also omitted the text overlays there.
